Sources/Commands/Options.swift

```diff
@@ -152,9 +152,20 @@ public struct SwiftToolOptions: ParsableArguments {
     @Option(help: "Specify the shared security directory")
     var securityPath: AbsolutePath?
 
-    /// Disables repository caching.
-    @Flag(name: .customLong("repository-cache"), inversion: .prefixedEnableDisable, help: "Use a shared cache when fetching repositories")
-    var useRepositoriesCache: Bool = true
+    /// Disables package caching.
+    @Flag(name: .customLong("dependencies-cache"), inversion: .prefixedEnableDisable, help: "Use a shared cache when fetching dependencies")
```

Comment:
not sure if `--disable-dependencies-cache` or `--disable-dependency-cache` is better

Reply:
Either is correct for an adjectival noun, but it looks as if most of the others use singular (e.g. the one you're replacing as well as things like `--enable-build-manifest-caching`).  So I'd say to go with `--disable-dependency-cache`.